cocos/editor-support/cocosbuilder/CCBAnimationManager.cpp

```diff
@@ -99,7 +99,11 @@ Node* CCBAnimationManager::getRootNode()
 
 void CCBAnimationManager::setRootNode(Node *pRootNode)
 {
+    CC_SAFE_RELEASE_NULL(_rootNode);
+    
     _rootNode = pRootNode;
+    
+    CC_SAFE_RETAIN(_rootNode);
```

Comment:
You retain the rootNode here, I think it should be release somewhere, like the destructor method, right?